Scoutsuite docs (#1013)
guardicore/monkey: docs/content/usage/integrations/scoutsuite.md

---
title: "Scoutsuite"
date: 2021-03-02T16:23:06+02:00
draft: false
description: "Scout Suite is an open-source cloud security-auditing tool."
weight: 10
---

### About ScoutSuite

<a href="https://github.com/nccgroup/ScoutSuite" target="_blank" >Scout Suite</a> is an open-source cloud security-auditing tool.
It queries the cloud API to gather configuration data. Based on configuration
data gathered, ScoutSuite shows security issues and risks present in your infrastructure.

### Supported cloud providers

> Currently, the Infection Monkey only supports AWS.

Reviewer: In the context of scoutsuite, this makes sense, but I wonder if someone may still find it a bit confusing. What about,

"Currently, the Infection Monkey can only use ScoutSuite to assess AWS environments."

Author: Reworded

    - If the command-line interface is available on the Island, it will be used to access 
    the cloud API
    
More details about configuring ScoutSuite can be found in the tool itself, by choosing 
"Cloud Security Scan" in the "Run Monkey" options. 

![Cloud scan option in run page](/images/usage/integrations/scoutsuite_run_page.png 
"Successful setup indicator")

After you're done with the setup, make sure that a checkmark appears next to the AWS option. This 
verifies that ScoutSuite can access the API.

![Successfull setup indicator](/images/usage/integrations/scoutsuite_aws_configured.png 
"Successful setup indicator")

### Running a cloud security scan

If you have successfully configured the cloud scan, Infection Monkey will scan
your cloud infrastructure when the Monkey Agent is run **on the Island**. You
can simply click on "From Island" in the run options to start the scan. The
scope of the network scan and other activities you may have configured the Agent
to perform are ignored by the ScoutSuite integration, except **Monkey
Configuration -> System info collectors -> AWS collector**, which needs to
remain **enabled**.

### Assessing scan results

> After the scan is done, ScoutSuite results will be sorted and applied to the ZeroTrust Extended framework 

Reviewer: ```suggestion
After the scan is done, ScoutSuite results will be categorized according to the ZeroTrust Extended framework 
```

and displayed as a part of the ZeroTrust report. The main difference between Infection Monkey findings and 
ScoutSuite findings is that ScoutSuite findings contain security rules. To see which rules were 

> checked click on the "Rules" button next to the relevant test. You'll see a brief overview of the rules 

Reviewer: ```suggestion
checked, click on the "Rules" button next to the relevant test. You'll see a brief overview of the rules 
```

> related to the test and a list of those rules. Expand a rule to see its description, remediation and 

Reviewer: "You'll see a brief overview of the rules related to the test and a list of those rules."

^ This is a little unclear. Why do we see an overview and a list? Is it a list of the rules that failed? Is it a list of all rules?

Maybe we can just get rid of "and a list of those rules" it it's redundant.

Author: Reworded

more details about resources flagged. Each flagged resource has a path so you can easily locate 

> them in the cloud and change the value that is deemed insecure.

Reviewer: ```suggestion
it in the cloud and remediate the issue.
```
